# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Washes your laundry
#
# goal message = the request
float64 number		# the number of items to wash
